src/rdkafka.c

```diff
@@ -676,6 +676,22 @@ static const struct rd_kafka_err_desc rd_kafka_err_descs[] = {
         _ERR_DESC(RD_KAFKA_RESP_ERR_FENCED_INSTANCE_ID,
                   "Broker: Static consumer fenced by other consumer with same "
                   "group.instance.id"),
+        _ERR_DESC(RD_KAFKA_RESP_ERR_ELIGIBLE_LEADERS_NOT_AVAILABLE,
+                  "Broker: Eligible topic partition leaders are not available"),
```

Comment:
I think the "topic" part is a bit redundant, and inconsistent with existing properties, even though the Java code has them I suggest stripping it out and just having "partition".